packages/core/core/src/index.ts

```diff
@@ -6,6 +6,7 @@ import { destroyOnSignal, resolveWorkingDirectories, createUpdateNotifier } from
 
 export { default as compileStrapi } from './compile';
 export * as factories from './factories';
+export { repairOrphanUnidirectionalRelations } from './services/document-service/utils/repair-orphan-unidirectional-relations';
```

Comment:
This probably isn't what we want to do 🤔. I'm not really sure the right place to put the function or the right pattern for exporting it. For now I just put it right next to the implementation and exported it here.

Reply:
See my comment above, it should be exported from database next to the original, but split. Sorry for the confusion I caused!